benchmarks/linear_programming/utils/get_datasets.py

```
@@ -707,17 +724,31 @@ def main():
         for bench in args.benchmarks:
-            for name in MittelmannInstances["benchmarks"][bench]:
-                download_dataset(name, instance_download_path)
-    return
+            datasets_to_download.extend(
+                MittelmannInstances["benchmarks"][bench]
+            )
+
+    for name in datasets_to_download:
+        try:
+            download_dataset(name, instance_download_path)
+        except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)